sway/commands/output/transform.c

```diff
@@ -10,30 +13,49 @@ struct cmd_results *output_cmd_transform(int argc, char **argv) {
 		return cmd_results_new(CMD_INVALID, "output",
 			"Missing transform argument.");
 	}
-
-	struct output_config *output = config->handler_context.output_config;
+	int32_t transform = 0;
 	if (strcmp(*argv, "normal") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_NORMAL;
+		transform = WL_OUTPUT_TRANSFORM_NORMAL;
 	} else if (strcmp(*argv, "90") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_90;
+		transform = WL_OUTPUT_TRANSFORM_90;
 	} else if (strcmp(*argv, "180") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_180;
+		transform = WL_OUTPUT_TRANSFORM_180;
 	} else if (strcmp(*argv, "270") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_270;
+		transform = WL_OUTPUT_TRANSFORM_270;
 	} else if (strcmp(*argv, "flipped") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_FLIPPED;
+		transform = WL_OUTPUT_TRANSFORM_FLIPPED;
 	} else if (strcmp(*argv, "flipped-90") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_FLIPPED_90;
+		transform = WL_OUTPUT_TRANSFORM_FLIPPED_90;
 	} else if (strcmp(*argv, "flipped-180") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_FLIPPED_180;
+		transform = WL_OUTPUT_TRANSFORM_FLIPPED_180;
 	} else if (strcmp(*argv, "flipped-270") == 0) {
-		output->transform = WL_OUTPUT_TRANSFORM_FLIPPED_270;
+		transform = WL_OUTPUT_TRANSFORM_FLIPPED_270;
 	} else {
 		return cmd_results_new(CMD_INVALID, "output",
 			"Invalid output transform.");
 	}
-
+	struct output_config *output = config->handler_context.output_config;
 	config->handler_context.leftovers.argc = argc - 1;
 	config->handler_context.leftovers.argv = argv + 1;
+
+	if (argc > 1 && (strstr(argv[1], "clockwise") != NULL)) {
```

Comment:
What happens if you execute `output XXX transform 90 asdfclockwise`? It should error out.

Reply:
Have altered to check specifically for `clockwise`/`anticlockwise`. I don't mind what these are called (left and right could be used, but using clockwise/anticlockwise seems to have the right amount of brevity and obviousness to me.